### DISCUSSION TITLE
Accelerometers from keyword file

### DISCUSSION THREAD

**Discussion by TAcertifix** [OpenRadioss/OpenRadioss#2357] [Q&A] [answered]
Hi there!

I've attempted to run the Yaris impact keyword in Openradioss, and while comparing the results, I noticed significant discrepancies in the acceleration values obtained from each accelerometer (in the TH CSV). These results diverge considerably from those documented in the verification & validation report, not even having the same order of magnitude. 
What could be causing this discrepancy? Is there anything specific I should add in the keyword file?

Thank you very much!

**Comment by PaulAltair** [comment]
Are you applying the same filter as in the CCSA published data? (e.g. CFC60) to your results? Can you share, a screenshot or specific accelerometer where you see a big difference, comparing here, OpenRadioss/DYNA CCSA report, all look similar? (Also the Dyna model is in mm/s^2, but CCSA report is in 'G' so you would need to / 9810 to get same units)
![image](https://github.com/OpenRadioss/OpenRadioss/assets/98418895/70282bd1-b119-43b4-ac39-b4da7fc03b5b)

**Reply by TAcertifix** [reply]
Good afternoon Paul, 

Thank you very much for your reply.

I haven't applied any filters yet.

Here are the results I'm obtaining for the TH-ACCEL 1 variable 8320 (1st column of the accelerometer output):
![image](https://github.com/OpenRadioss/OpenRadioss/assets/160721383/ae780554-8c41-4f3c-b7de-3e6ac8ab3ad0)

And, here are the results for the TH node of the accelerometer (7th column of the TH node output):
![image](https://github.com/OpenRadioss/OpenRadioss/assets/160721383/a57c70c1-d0c8-489b-8464-71cc2ff3598f)

**Reply by PaulAltair** [reply]
From my data, they do not correspond to anything like the values I see, your 1st plot looks a little like resultant velocity in shape, but even then, the values are wrong, maybe it is related to the th-csv conversion? (I was reading the T01 directly, if I convert to csv, the fields match up ok as below:) 
![image](https://github.com/OpenRadioss/OpenRadioss/assets/98418895/7770d03d-99bb-4221-ac7f-5d61758974ef)

**Reply by TAcertifix** [reply]
Good afternoon Paul,

Thank you very much for your help.

You were right. In the end the problem was related to the software I was using to read the CSV file. When I imported the T01 values into Paraview, somehow I messed up the data and the results didn't match the header of the column. 

I've just substracted the accelerometer data using a python script and the new results make a lot more sense.
I guess the results are not exactly the same because I'm running the Toyota Yaris coarse model and not the detailed one, but the plot is simmilar to the one you shared:
![image](https://github.com/OpenRadioss/OpenRadioss/assets/160721383/0b0a7ff3-0d24-44b9-8426-da63c5c9cbef)


Once again, thank you!

**Comment by PaulAltair** [comment]
one other suggestion, if you run only the 'starter' first, then put '/TH/TITLE' in the generated '0001.rad' before running the engine, you then get the variable names in the csv file that is converted, which makes it a little easier to trace what is what: i.e. you then get 'AX' rather than 'Var nnnn' 

I have attached the first 50ms of my run csv here so you can compare to yours (the full file is too large to attach here even zipped unfortunately)

[YarisOpenRadiossT01_1.zip](https://github.com/OpenRadioss/OpenRadioss/files/14390787/YarisOpenRadiossT01_1.zip)

I extracted just the ACCEL_1 AX, for the full run too:

[YarisOpenRadiossT01_ACCEL1.csv](https://github.com/OpenRadioss/OpenRadioss/files/14390849/YarisOpenRadiossT01_ACCEL1.csv)

![image](https://github.com/OpenRadioss/OpenRadioss/assets/98418895/5beb21a8-0b54-479c-bed1-bf294398f2d8)

**Reply by TAcertifix** [reply, marked as answer]
Thank you very much for the suggestion and the data!
